etc/makefile

```diff
@@ -1,4 +1,5 @@
-CFLAGS += -Wall -W -Wextra -Wshadow -O3 -I../
+LTM_CFLAGS += -Wall -W -Wextra -Wshadow -O3 -I../
```

Comment:
Why complicating things here? the user could already add flags to CFLAGS, couldn't they?

Reply:
yeah but only through `CFLAGS="-foo" make` now it's also possible via `make CFLAGS="-foo"`

Reply:
Ok